modules/localstack/build.gradle

```diff
@@ -5,4 +5,7 @@ dependencies {
 
     compileOnly 'com.amazonaws:aws-java-sdk-s3:1.11.479'
     testCompile 'com.amazonaws:aws-java-sdk-s3:1.11.479'
+
+    compileOnly 'com.amazonaws:aws-java-sdk-sqs:1.11.479'
```

Comment:
Is this needed as `compileOnly`?
Probably only `testCompile` necessary.

Reply:
No, it's not needed but then also I would question `compileOnly` dependency to s3. I would change it to `aws-java-sdk-core` then